CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+Features:
+* UBI-based images now built and published with releases: [GH-288](https://github.com/hashicorp/vault-secrets-operator/pull/288)
+* `ppc64le` and `s390x` builds are now included in releases: [GH-288](https://github.com/hashicorp/vault-secrets-operator/pull/288)
```

Comment:
I couldn't decide between Changes or Features for these, so definitely open to suggestions.